### ISSUE TITLE
ser_predictor返回的label与ocr的不同

### ISSUE THREAD

**Issue by eerenyuan** [PaddlePaddle/PaddleOCR#8648]
我参考kie\predict_kie_token_ser.py文件生成了一个SerPredictor，其中的ser是我自己训练的，det和rec用的pretrained_en。

先生成了一个参数列表
parser = init_args()
args=parser.parse_args(['--kie_algorithm=LayoutXLM',
  '--ser_model_dir=f:/pan/ser_vi_layoutxlm',
    '--ser_dict_path=f:/pan/pan_class_list.txt',
    '--vis_font_path=../doc/fonts/simfang.ttf',
    '--ocr_order_method=tb-yx',
    '--use_angle_cls=True'
])

后面这一段基本是从predict_kie_token_ser.py里面照搬的，只是把语言改成了en。

import cv2
import json
import numpy as np
import time

import paddleocr.tools.infer.utility as utility
from ppocr.data import create_operators, transform
from ppocr.postprocess import build_post_process
from ppocr.utils.logging import get_logger
from ppocr.utils.visual import draw_ser_results
from ppocr.utils.utility import get_image_file_list, check_and_read
from ppstructure.utility import parse_args

from paddleocr import PaddleOCR

logger = get_logger()

class SerPredictor(object):
    def __init__(self, args):
        self.ocr_engine = PaddleOCR(
            lang="en",
            use_angle_cls=args.use_angle_cls,
            det_model_dir=args.det_model_dir,
            rec_model_dir=args.rec_model_dir,
            show_log=False,
            use_gpu=args.use_gpu)

        pre_process_list = [{
            'VQATokenLabelEncode': {
                'algorithm': args.kie_algorithm,
                'class_path': args.ser_dict_path,
                'contains_re': False,
                'ocr_engine': self.ocr_engine,
                'order_method': args.ocr_order_method,
            }
        }, {
            'VQATokenPad': {
                'max_seq_len': 512,
                'return_attention_mask': True
            }
        }, {
            'VQASerTokenChunk': {
                'max_seq_len': 512,
                'return_attention_mask': True
            }
        }, {
            'Resize': {
                'size': [224, 224]
            }
        }, {
            'NormalizeImage': {
                'std': [58.395, 57.12, 57.375],
                'mean': [123.675, 116.28, 103.53],
                'scale': '1',
                'order': 'hwc'
            }
        }, {
            'ToCHWImage': None
        }, {
            'KeepKeys': {
                'keep_keys': [
                    'input_ids', 'bbox', 'attention_mask', 'token_type_ids',
                    'image', 'labels', 'segment_offset_id', 'ocr_info',
                    'entities'
                ]
            }
        }]
        postprocess_params = {
            'name': 'VQASerTokenLayoutLMPostProcess',
            "class_path": args.ser_dict_path,
        }

        self.preprocess_op = create_operators(pre_process_list,
                                              {'infer_mode': True})
        self.postprocess_op = build_post_process(postprocess_params)
        self.predictor, self.input_tensor, self.output_tensors, self.config = \
            utility.create_predictor(args, 'ser', logger)

    def __call__(self, img):
        ori_im = img.copy()
        data = {'image': img}
        data = transform(data, self.preprocess_op)
        if data[0] is None:
            return None, 0
        starttime = time.time()

        for idx in range(len(data)):
            if isinstance(data[idx], np.ndarray):
                data[idx] = np.expand_dims(data[idx], axis=0)
            else:
                data[idx] = [data[idx]]

        for idx in range(len(self.input_tensor)):
            self.input_tensor[idx].copy_from_cpu(data[idx])

        self.predictor.run()

        outputs = []
        for output_tensor in self.output_tensors:
            output = output_tensor.copy_to_cpu()
            outputs.append(output)
        preds = outputs[0]

        post_result = self.postprocess_op(
            preds, segment_offset_ids=data[6], ocr_infos=data[7])
        elapse = time.time() - starttime
        return post_result, data, elapse

ser_predictor = SerPredictor(args)

这个ser_predictor对象里面包含一个ocr_engine

我发现直接调用ser_predictor.ocr_engine.ocr(img)会返回较多的label（没有分类）

'[[[[807.0, 96.0], [967.0, 107.0], [965.0, 132.0], [805.0, 121.0]], ["atSignature", 0.825221598148346]], [[[1113.0, 121.0], [1222.0, 125.0], [1221.0, 149.0], [1112.0, 145.0]], ["29/04/2000", 0.9971332550048828]], [[[1115.0, 143.0], [1225.0, 147.0], [1224.0, 170.0], [1114.0, 166.0]], ["Date of Birth", 0.9448187947273254]], [[[429.0, 182.0], [507.0, 182.0], [507.0, 200.0], [429.0, 200.0]], ["30112026", 0.953766942024231]], [[[965.0, 208.0], [1231.0, 213.0], [1231.0, 237.0], [965.0, 231.0]], ["MANUBALLA MANI BABU", 0.9606010913848877]], [[[980.0, 226.0], [1233.0, 235.0], [1232.0, 258.0], [979.0, 250.0]], ["T /Father\'s Name", 0.8446358442306519]], [[[1016.0, 276.0], [1238.0, 282.0], [1237.0, 305.0], [1016.0, 300.0]], ["MANUBALLA SURAZ", 0.9648517370223999]], [[[1137.0, 298.0], [1241.0, 304.0], [1240.0, 327.0], [1136.0, 321.0]], ["TName", 0.9382625818252563]], [[[773.0, 365.0], [992.0, 370.0], [992.0, 398.0], [772.0, 393.0]], ["NPRPS8427E", 0.9968595504760742]], [[[698.0, 411.0], [1071.0, 417.0], [1070.0, 445.0], [698.0, 440.0]], ["Permanent Account Number Card", 0.9800944924354553]], [[[413.0, 493.0], [710.0, 500.0], [709.0, 533.0], [412.0, 526.0]], ["GOVT.OFINDIA", 0.9662520289421082]], [[[909.0, 501.0], [1257.0, 508.0], [1256.0, 543.0], [908.0, 536.0]], ["INCOME TAX DEPARTMENT", 0.9516409039497375]], [[[404.0, 543.0], [565.0, 548.0], [564.0, 583.0], [403.0, 578.0]], ["diepdt", 0.619767963886261]], [[[589.0, 546.0], [715.0, 552.0], [713.0, 589.0], [587.0, 583.0]], ["PXIH", 0.7324607968330383]], [[[909.0, 544.0], [1054.0, 554.0], [1051.0, 601.0], [906.0, 591.0]], ["FEHTT", 0.6281695365905762]], [[[1078.0, 554.0], [1251.0, 558.0], [1250.0, 593.0], [1077.0, 590.0]], ["ephlte", 0.5564804673194885]]]'

而使用
ser_res, _, elapse = ser_predictor(img)
只会得到很少的label，且内容完全读不通。

[[{'transcription': '221000000',
   'bbox': [1112, 121, 1222, 149],
   'points': [[1113.0, 121.0],
    [1222.0, 125.0],
    [1221.0, 149.0],
    [1112.0, 145.0]],
   'pred_id': 0,
   'pred': 'O'},
  {'transcription': 'diepdt',
   'bbox': [403, 543, 565, 583],
   'points': [[404.0, 543.0], [565.0, 548.0], [564.0, 583.0], [403.0, 578.0]],
   'pred_id': 0,
   'pred': 'O'},
  {'transcription': 'PXIH',
   'bbox': [587, 546, 715, 589],
   'points': [[589.0, 546.0], [715.0, 552.0], [713.0, 589.0], [587.0, 583.0]],
   'pred_id': 0,
   'pred': 'O'},
  {'transcription': 'ephlte',
   'bbox': [1077, 554, 1251, 593],
   'points': [[1078.0, 554.0],
    [1251.0, 558.0],
    [1250.0, 593.0],
    [1077.0, 590.0]],
   'pred_id': 0,
   'pred': 'O'}]]

这样的结果合理吗？ 
我能不能强制要求ser_predictor使用ocr的结果？ 

似乎infer_kie_token_ser.py里面有infer_mode这个参数，但是predict_kie_token_ser.py中没有，这两个文件的区别是什么？

谢谢。
